if.json

```diff
@@ -24,55 +24,55 @@
             "required": true
         },
         "accept": {
-            "description": "A value that is returned if the boolean expression is `true`. Defaults to `true`.",
+            "description": "A value that is returned if the boolean value is `true`. Defaults to `true`.",
             "schema": {
                 "description": "Any data type is allowed."
             },
             "default": true
         },
         "reject": {
-            "description": "A value that is returned if the boolean expression is `false`. Defaults to `false`.",
+            "description": "A value that is returned if the boolean value is `false`. Defaults to `false`.",
```

Comment:
I'm wondering if there would be more useful options for the default values of accept and reject. Now with true/false as defaults, it would make the `if` process redundant in a way (`if(x)` would be just `x` basically).

Another option would be to use `null` as default for reject, but I'm not sure what a good default for accept would be then (int `1` maybe?)

Reply:
@soxofaan Indeed, I think my original intention was to allow expressions/conditions (e.g. eq(foo, bar)) in if, but the way it is now it's really just a not so useful boolean to boolean mapper by default. Do we even need a default for accept? Maybe remove the default for accept and then set the default for reject to null?

Reply:
> Maybe remove the default for accept and then set the default for reject to null?

that sounds very sensible indeed

Reply:
Done.